src/Microsoft.Sbom.Targets/Microsoft.Sbom.Targets.targets

```diff
@@ -70,20 +76,9 @@
       <Output TaskParameter="SbomPath" PropertyName="SbomPathResult" />
     </GenerateSbom>
     <Message Importance="High" Text="Task result: $(SbomPathResult)" />
-  </Target>
 
-  <!-- Specify the SBOM files to copy into the nuget package -->
-  <Target Name="CopySbomOutput" DependsOnTargets="GenerateSbomTarget">
-    <PropertyGroup>
-      <!--When building frameworks such as net8.0-windows, the platform version is appended to the framework in the NuGet package-->
-      <TargetFrameworkWithPlatformVersion Condition="$(TargetPlatformVersion) != ''">$(TargetFramework)$(TargetPlatformVersion)</TargetFrameworkWithPlatformVersion>
-      <TargetFrameworkWithPlatformVersion Condition="$(TargetPlatformVersion) == ''">$(TargetFramework)</TargetFrameworkWithPlatformVersion>
-    </PropertyGroup>
-    <ItemGroup>
-      <!--Add manifest and SHA file from the GenerateSbom target execution-->
-      <TfmSpecificPackageFile Include="$([System.IO.Path]::Combine($(SbomPathResult),$(SbomSpecification)))\**">
-        <PackagePath>$([System.IO.Path]::Combine($(BuildOutputTargetFolder),$(TargetFrameworkWithPlatformVersion),$(ManifestFolderName),$(SbomSpecification)))</PackagePath>
-      </TfmSpecificPackageFile>
-    </ItemGroup>
+    <!-- Zip the Nuget package back up and delete the temporary unzipped package. -->
+    <ZipDirectory SourceDirectory="$(NugetPackageUnzip)" DestinationFile="$(NugetPackage)" Overwrite="true" />
```

Comment:
Out of curiosity, about how much longer does the unzip/zip process take compared to our original method?

Reply:
I have updated the description with the performance:

### .NET Standard project
In average, the workaround has a better performance than the original solution. I ran some performance tests using **a .NET Standard** project we use in production:
| | Generate SBOM per framework | Generate SBOM per package (with workaround) |
| -------- | ------- | -------- |
| Avg. Time | 2.95 | 2.85 |

However, in one of the performance tests, there was one outlier of 3.767 seconds for the workaround, probably due to a cold-start.

### Multi-framework project
I also ran some performance tests using a dummy multi-framework project:
| | Generate SBOM per framework | Generate SBOM per package (with workaround) |
| -------- | ------- | -------- |
| Avg. Time | 3.15 | 3.16 |

However, there were two cold-start runs that gave outlier results:
| | Generate SBOM per framework | Generate SBOM per package (with workaround) |
| -------- | ------- | -------- |
| Cold-Start Time | 4.98 | 4.22 |